applications/nrf_desktop/doc/settings_load.rst

```diff
@@ -0,0 +1,30 @@
+.. _settings_load:
+
+Settings load module
+####################
+
+Use the Settings load module to trigger loading the data from non-volatile memory.
+
+Module Events
+*************
+
++----------------+-------------+-------------------+-----------------+------------------+
+| Source Module  | Input Event | This Module       | Output Event    | Sink Module      |
++================+=============+===================+=================+==================+
+|                |             | ``settings_load`` |                 |                  |
++----------------+-------------+-------------------+-----------------+------------------+
+
+Configuration
+*************
+
+The Settings load module is enabled for every nRF Desktop device with :ref:`zephyr:settings` enabled - when :option:`CONFIG_SETTINGS` is set.
+
+Loading all the settings in one go is more efficitent than making each subsystem load the settings independently.
+Zephyr assumes that the application is responsible for calling the :cpp:func:`settings_load` after having done all necessary initialization.
```

Comment:
It may be worth adding a list of modules we are waiting for. Or maybe we should create settnigs_load_def.h file when such list is given?

Reply:
(most probably no)

Reply:
I think we could refer to this module from other modules that use Zephyr settings (I did it already in the Bluetooth guide)